nav2_controller/src/controller_server.cpp

```diff
@@ -667,41 +675,55 @@ void ControllerServer::computeAndPublishVelocity()
   RCLCPP_DEBUG(get_logger(), "Publishing velocity at time %.2f", now().seconds());
   publishVelocity(cmd_vel_2d);
 
-  // Find the closest pose to current pose on global path
-  geometry_msgs::msg::PoseStamped robot_pose_in_path_frame;
-  if (!nav2_util::transformPoseInTargetFrame(
-          pose, robot_pose_in_path_frame, *costmap_ros_->getTfBuffer(),
-          current_path_.header.frame_id, costmap_ros_->getTransformTolerance()))
-  {
-    throw nav2_core::ControllerTFError("Failed to transform robot pose to path frame");
-  }
-
-  std::shared_ptr<Action::Feedback> feedback = std::make_shared<Action::Feedback>();
-  feedback->speed = std::hypot(cmd_vel_2d.twist.linear.x, cmd_vel_2d.twist.linear.y);
-
-  nav_msgs::msg::Path & current_path = current_path_;
-  auto find_closest_pose_idx = [&robot_pose_in_path_frame, &current_path]()
+  double current_distance_to_goal_ = 0.0;
+  nav2_msgs::msg::TrackingFeedback current_tracking_feedback_;
```

Comment:
```suggestion
  nav2_msgs::msg::TrackingFeedback current_tracking_feedback;
```

This is a local member, there shouldn't be an underscore